### DISCUSSION TITLE
"Create new" popup menu is stuck open

### DISCUSSION THREAD

**Discussion by VictorWesterlund** [community/community#57229] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have an issue with the card intended to appear only when the "+" button in the menu bar is clicked, remains open continuously. This problem has persisted for several weeks, and clicking the "+" button does not trigger any response. Notably, this issue is not specific to any particular device and persists across sessions.

![image](https://github.com/community/community/assets/35688133/101a0fdd-a63b-4ac9-8a63-c81b9b777db4)
![image](https://github.com/community/community/assets/35688133/94799ae8-17c4-451a-9a4c-e096d55dfff6)

**Comment by semmakara6666** [comment]
18:16

tel. 6

Google Drive

Google will review your request

Thank you for reporting that this content may violate our Terms of Use.
Your report has been submitted and will be processed as soon as possible.

© 2023 Google - Help “Privacy & Terms”

Google Drive - Successfully Reported abuse.com google.com

នៅ​ថ្ងៃទី ចន្ទ 5 មិថុនា 2023, 18:30 Victor Westerlund <
***@***.***> បាន​សរសេរ​ថា៖

**Comment by VictorWesterlund** [comment]
Latest release fixed this issue. Thanks!

**Reply by gbozo** [reply]
nope, still here

**Reply by dchasman** [reply]
I am now also experiencing this issue in Edge on iPadOS. 

![IMG_0069](https://github.com/community/community/assets/16857/e6cb9208-a0b6-4feb-9776-4286876ab5ae)


First noticed it this morning. It survives clearing cache, using the menu to attempt to create a Gist, and is extremely challenging to work around because it obscures valid areas of the page entirely.

Current workaround is to use Chrome - so far the issue has not manifested there.

**Reply by VictorWesterlund** [reply]
Strange.. the issue has not resurfaced for me (yet). Perhaps it would be best to open a new issue for this. I would not be able to follow-up on feedback or suggestions since I no longer have this problem.

**Reply by dchasman** [reply]
So far I am only able to repro the issue on iPadOS 17 using Edge - wacky. Just working around that combination as I now suspect the issue is in the browser not the site.

**Reply by VictorWesterlund** [reply]
I was able to repro the issue in both Chrome and Firefox on two separate devices at the time - if I recall. Not sure if it's related to what you're experiencing tho since my issue persisted across sessions, browsers, and devices. I agree your issue sounds local/browser based and not server-side, but that's only my guess.
